examples/gno.land/p/demo/ufmt/ufmt.gno

```diff
@@ -48,6 +48,16 @@ func Sprintf(format string, args ...interface{}) string {
 			default:
 				buf += "(unhandled)"
 			}
+		case 't':
+			if v, ok := arg.(bool); ok {
+				if v == true {
```

Comment:
Nitpick:

`if v {...}`, no need for `true`

Reply:
Shouldn't this be detected by a linter?

Reply:
Probably by revive (which is essentially golint v2), which I don't think is enabled. On top of it, gno files are still not properly linted

Reply:
Yes, I believe [`gosimple`](https://github.com/dominikh/go-tools/tree/master/simple) also provides this check

Reply:
Resolving because of [beeabea](https://github.com/gnolang/gno/pull/574/commits/beeabea8ed26e35dfb667dda58b99dada70ded78)